Products/ZenUI3/browser/resources/js/zenoss/form/graphPanel.js

```diff
@@ -182,6 +188,11 @@
                     xtype: 'toolbar',
                     dock: 'top',
                     items: ['->',{
+                        text: _t('Open in New Tab'),
+                        handler: Ext.bind(function(btn, e) {
```

Comment:
This isn't necessary but it can be rewritten as 

Ext.bind(this.newTab, this);